### PR TITLE
Depth calibration tutorial

### PR DESCRIPTION
**Pull request by cretaceous-creature**
add depth calibration tutorial............

### CODE REVIEW

File: jsk_pcl_ros/README.md [hunk 1]
```diff
@@ -1634,6 +1634,55 @@ Plug the depth sensor which can be launched by openni.launch and run the below c
 roslaunch jsk_pcl_ros tf_transform_cloud.launch
 ```
 
+
+## Depth Camera Calibration(Kinect,Xtion,Primesense)
+![](images/depth_calibration.png)
+### Two Main Steps:
+
+*  `1. Camera Intrisic Calibration: Intrisic Calibration` `Please refer to ros RGB camera calibration tutorial` >> `http://wiki.ros.org/camera_calibration/Tutorials/MonocularCalibration`
```
Comment (garaemon) [inline jsk_pcl_ros/README.md]: Please use link markup
```
[string](url)
``

It will be [string](https://google.com)

File: jsk_pcl_ros/README.md [hunk 2]
```diff
@@ -1634,6 +1634,55 @@ Plug the depth sensor which can be launched by openni.launch and run the below c
...
+
+*  `2. Depth Calibration: Depth Calibration` related to the distance z, optical frame pixel u v and global pixel offset.
+
+### You need:
+   
+*   Chessboard: Make sure ur Application Range, if u wanna use camera in short range(0.5m~2m) choose small chessboard, otherwise plz choose a larger one. 
+*   Depth Camera: Kinect one, Xtion, Primesense. (There may be some problem when using primesense, check `http://answers.ros.org/question/197318/openni2_launch-doesnt-work-with-carmine-109-connected-to-usb30/?comment=198397#comment-198397` to install the newest openni2, perhaps u need to do `apt-get remove libopenni2-0` first)
```
Comment (garaemon) [inline jsk_pcl_ros/README.md]: link markup

File: jsk_pcl_ros/README.md [hunk 3]
```diff
@@ -1634,6 +1634,55 @@ Plug the depth sensor which can be launched by openni.launch and run the below c
...
+*   Good PC with ubuntu and ros installed:  We only tested in Lenovo thinkpad series.
+*   jsk\_pcl\_ros:   `https://github.com/jsk-ros-pkg/jsk_recognition/tree/master/jsk_pcl_ros`
+
+### Camera Intrisic Calibration:
+
+* Please follow `http://wiki.ros.org/camera_calibration/Tutorials/MonocularCalibration` tutorial and when u finished calibration(4 features become green), wait patiently until u can click upload, calibration file will be right there in `~/.ros/camera_info/***.yaml` waiting.  Check the `openni2_launch.launch` file or `openni2_local.launch` if u use JSK package to edit the path.
```
Comment (garaemon) [inline jsk_pcl_ros/README.md]: link markup

**Comment by garaemon** [comment]
great! please update your commit according to my line notes

**Comment by cretaceous-creature** [comment]
Committed....
sorry for replying late......
風邪を引いちゃった。。。。

2015-02-03 12:11 GMT+09:00 Ryohei Ueda <notifications@github.com>:

**Pull request merged**